src/quo/components/settings/category/reorder/view.cljs

```diff
@@ -16,9 +16,9 @@
   (reagent/flush))
 
 (defn- reorder-category-internal
-  [{:keys [label data blur? theme]}]
+  [{:keys [label data blur? theme container-style]}]
   (reagent/with-let [atom-data (reagent/atom data)]
-    [rn/view {:style style/container}
+    [rn/view {:style (merge (style/container label) container-style)}
```

Comment:
FYI: `style/container` is a function. I just made this component identical in behavior to its cousin `settings.category.settings.view`.